components/Projects/FiltersBar/FiltersBar.tsx

```diff
@@ -85,12 +87,13 @@ const FiltersBar: React.FC = () => {
                 <>
                   <div
                     ref={filterRef}
-                    className="z-[101] absolute -bottom-[23.5rem] md:-bottom-[14rem] right-2 px-[34px] py-[27px] rounded-md border border-blue-600 min-w-[400px] md:min-w-0 md:w-[863px] h-[360px] md:h-[209px] p-5 bg-gray-50 dark:bg-gray-600"
+                    className="z-[101] absolute -bottom-[23.5rem] md:-bottom-[14rem] right-2 px-[34px] py-[27px] rounded-md border border-blue-600 min-w-[400px] md:min-w-0 md:w-[863px] min-h-[260px] md:h-[209px] p-5 bg-gray-50 dark:bg-gray-600"
                   >
                     <div className="flex flex-col gap-[22px]">
                       <h3 className="text-base font-bold leading-normal">
                         פילטרים לפרויקטים
                       </h3>
+                      {/* sort by set 1 */}
                       <div className="flex gap-4 md:gap-[26px] justify-center md:justify-normal md:items-center">
```

Comment:
How about instead of a comment explaining the code extract this code chunk into a custom component (let`s say `SortX`) and use it instead? 

I think it will be easier for other developers to read this code, also it will make this file smaller which is nice

Reply:
this filtering is not yet implemented.